src/survey/views.py

```diff
@@ -12,8 +12,9 @@
 from .serializers import SurveySerializer
 
 
-class SurveysView(generics.GenericAPIView):
+class AllSurveysView(generics.GenericAPIView):
     serializer_class = SurveySerializer
+    # TODO @njs99: figure out the best way to limit this to superusers
```

Comment:
wouldn't all you have to do is set the permission classes to `api_settings.ADMIN_PERMISSIONS` ?

Reply:
yea u right lol, i thought i had ran into issues with that before but I just double-checked and its working on `/api/matches/` for something similar